app/main/views/services.py

```diff
@@ -41,6 +43,10 @@ def list_services():
 
     services = Service.query.filter(
         Service.framework.has(Framework.expired == false())
+    ).order_by(
```

Comment:
Actually, I think the point does make sense - why do we want ordering on the "fetch all services" endpoint by default? JSON isn't supposed to be ordered...

If the story is the supplier page then perhaps ordering should only be applied to queries with a supplier - i.e. in the `if supplier_id is not None:` part of the method.